src/expressions.rst

```diff
@@ -3266,6 +3270,29 @@ The :t:`evaluation` of a :t:`method call expression` proceeds as follows:
    let click = ClickCommand { ... };
    click.execute();
 
+.. _fls_8gPCPVc99pXJ:
+
+Call Conformance
+~~~~~~~~~~~~~~~~
+
+:dp:`fls_tsn6SUUG9LvW`
+A :t:`method call expression` is equivalent to a :t:`call expression` where the
+:t:`call operand` is the resolved :t:`method` and the adjusted :t:`receiver
+operand` prepended to the :t:`[argument operand]s`.
+
+:dp:`fls_c40C6rg6rGv6`
+An :t:`argument operand` matches a :t:`function parameter` or :t:`field` of the :t:`callee type` when its position and the position of the
+:t:`parameter` or :t:`field` of the :t:`callee type` are the same. Such an
```

Comment:
No need to repeat "of the callee type" because we already identified which function parameter and field we are talking about.

Note that :t:`parameter` should be :t:`function parameter`.